js/up.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+const form = document.forms.userProfile;
+const ageInput = document.querySelector('#age').value;
+const heightInput = document.querySelector('#height').value;
+const weightInput = document.querySelector('#weight').value;
+const bmiInput = document.querySelector('#bmi').value;
+const cityInput = document.querySelector('#city').value;
+const stateInput = document.querySelector('#state').value;
+const fitnessGoalsInput = document.querySelector('#fitnessGoals').value;
+
+console.log(ageInput);
+
+// function populateLocalStorage() {
```

Comment:
Abby, you need to use the variable, i.e., ageInput not in quotes as the value for this to work.

Reply:
Nevermind, that does not work.